### DISCUSSION TITLE
This node requires Node-RED custom integration version 1.1.0+  to be installed in Home Assistant for it to function.

### DISCUSSION THREAD

**Discussion by aperezva** [zachowj/node-red-contrib-home-assistant-websocket#1506] [General]
After update to 0.68.1 in the pallette, node-red-contrib-home-assistant-websocket I have this error:

This node requires Node-RED custom integration version 1.1.0+  to be installed in Home Assistant for it to function.

**Comment by zachowj** [comment]
Which node? What does the diagnostics button show for the installed companion version?

<https://zachowj.github.io/node-red-contrib-home-assistant-websocket/guide/diagnostics.html>

**Reply by aperezva** [reply]
A sensor entity,

I dont have the option to copy home assistant companion, but in my integration I have 4.0.2 version I it was working fine till update the websocket

<img width="605" alt="Captura de pantalla 2024-08-28 080536" src="https://github.com/user-attachments/assets/382a30da-9b8a-4059-8557-3f9bcfdbb732">

**Reply by aperezva** [reply]
More information. If I access directly the Ip:1880 I can see that is working. But in the node-red addon in home assistant directly i can´t see them

**Reply by zachowj** [reply]
Similar issues have been reported across the community. This doesn't seem to be a problem within this package but is likely related to the add-on or accessing it through an iframe.
